cargo-near/src/util/tracking.rs

```diff
@@ -0,0 +1,85 @@
+use reqwest::Client;
+use std::{env, process::Command, str};
+use tracing::debug;
+
+const SEND_TRACKING_REQUEST_ERROR: &str = "Can't send tracking usage event";
+
+#[derive(Debug, serde::Serialize)]
+struct MixpanelProperties {
+    token: String,
+    cli_version: String,
+    os: String,
+}
+
+#[derive(Debug, serde::Serialize)]
+struct TrackingData {
+    event: String,
+    properties: MixpanelProperties,
+}
+
+fn trim_newline(s: String) -> String {
+    let mut result = s;
+
+    if result.ends_with('\n') {
+        result.pop();
+        if result.ends_with('\r') {
+            result.pop();
+        }
+    }
+
+    result
+}
+
+pub(crate) fn track_usage() {
+    let output = if cfg!(target_os = "windows") {
+        Command::new("cmd")
+            .args(["/C", "near --version"])
+            .output()
+            .expect("failed to execute process")
+    } else {
+        Command::new("sh")
+            .arg("-c")
+            .arg("near --version")
+            .output()
+            .expect("failed to execute process")
+    };
+
+    let cli_version: String = match str::from_utf8(&output.stdout) {
```

Comment:
Version of `cargo-near` can be addeded by using `env!("CARGO_PKG_VERSION")`.
It's very strange to try to track `near` version, which even may not be installed and is not required to be installed, to measure the count of `cargo near new` calls.

Reply:
Thank you for the explanation, I changed it.
I was confused by asking to track CLI version. CLI in my mind is `near-cli-rs` :)